docs/reference/commandline/dockerd.md

```diff
@@ -396,61 +396,241 @@ Defaults to 20G.
+  `PATH`.
+
+`options` is optional. It lets you specify the runtime configuration that you
+want to use for the shim. The configuration parameters that you can specify in
+`options` depends on the runtime you're registering. For most shims,
+the supported configuration options are `TypeUrl` and `ConfigPath`.
+For example:
+
+```json
+{
+  "runtimes": {
+    "gvisor": {
+      "runtimeType": "io.containerd.runsc.v1",
+      "options": {
+        "TypeUrl": "io.containerd.runsc.v1.options",
+        "ConfigPath": "/etc/containerd/runsc.toml",
+      }
+    }
+  }
+}
+```
+
+You can configure multiple runtimes using the same runtimeType. For example:
+
+```json
+{
+  "runtimes": {
+    "gvisor-foo": {
+      "runtimeType": "io.containerd.runsc.v1",
+      "options": {
+        "TypeUrl": "io.containerd.runsc.v1.options",
+        "ConfigPath": "/etc/containerd/runsc-foo.toml"
+      }
     },
-    "runc": {
-      "path": "runc"
+    "gvisor-bar": {
+      "runtimeType": "io.containerd.runsc.v1",
+      "options": {
+        "TypeUrl": "io.containerd.runsc.v1.options",
+        "ConfigPath": "/etc/containerd/runsc-bar.toml"
+      }
     }
   }
 }
 ```
 
-This is the same example via the command line:
+If you want to use the runc shims with custom configuration, note that the
+`options` field takes a special set of configuration parameters when used with
+runc shims. For more information about runc parameters, refer to the runc
+configuration section in
+[CRI Plugin Config Guide](https://github.com/containerd/containerd/blob/main/docs/cri/config.md#full-configuration).
+
+###### Configure runc drop-in replacements
+
+If the runtime that you want to register can act as a drop-in replacement for
+runc, use the following format for the runtime entry:
+
+```json
+{
+  "runtimes": {
+    "<runtime>": {
+      "path": "/path/to/binary",
+      "runtimeArgs": ["...args"]
+    }
+  }
+}
+```
+
+Entries with this format use the default containerd runc shim to invoke a
+custom runtime binary, with optional CLI args. For an example, see
+[Alternative container runtimes > youki](https://docs.docker.com/engine/alternative-runtimes/#youki)
+
+##### Register runtimes using the CLI
+
+You can use the `--add-runtime` CLI flag to register runtimes on daemon startup.
+
+You can specify either the name of a fully qualified containerd runtime shim,
+or the name of a registered runtime. The executable must be installed on `PATH`.
+
+The following example adds a runtime named `youki`:
 
 ```console
-$ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-runc-replacement
+$ sudo dockerd --add-runtime youki=/usr/bin/youki
 ```
 
 > **Note**
 >
 > Defining runtime arguments via the command line is not supported.
 
-#### Options for the runtime
+##### Configure the default container runtime
+
+You can specify either the name of a fully qualified containerd runtime shim,
+or the name of a registered runtime. The executable must be installed on `PATH`.
```

Comment:
The default runtime option sets the default value for the `docker run --runtime` flag, and therefore has exactly the same acceptable range of values. That is, the default runtime can be set to either
1. the name of a registered runtime, or
2. the fully-qualified name of a shim whose executable is installed on `PATH`.

As currently phrased, it could be read that _a registered runtime name_ can only be set as the default runtime when the runtime executable is installed on `PATH`, which is not true. If it's registered, it can be set as the default, full stop.